hsctl

```diff
@@ -419,10 +423,48 @@ reset_all_hs() {
     if [[ -f schema.xml ]]; then rm -f schema.xml; fi
 }
 
+dis_build() {
+# Start Docker container and Run build
+
+docker run -i -v ~/hydroshare:/hydroshare --name=nodejs node:latest /bin/bash << eof
+
+cd hydroshare
+sleep 1
+cd hs_discover
+rm -rf static templates
+mkdir -p static templates/hs_discover
+npm install
+sleep 1
+npm run build
+sleep 1
+mkdir -p static/js
+mkdir -p static/css
+sleep .5
+cp -rp templates/hs_discover/js static/
+cp -rp templates/hs_discover/css static/
+cp -p templates/hs_discover/map.js static/js/
+echo "----------------js--------------------"
+ls -l static/js
+echo "--------------------------------------"
+sleep 1
+echo "----------------css-------------------"
+ls -l static/css
+echo "--------------------------------------"
+sleep 1
+cd static/
+mv js/app.*.js js/app.js
+mv js/chunk-vendors.*.js js/chunk-vendors.js
+cd ..
+eof
+
+docker container rm nodejs
+sleep 1
+}
+
```

Comment:
I thought there had to be a permissions change on the folder here?

Reply:
Because of the placement of the call, the permission changes were already being done in a later step of the rebuild.